src/content/homepage/charts/second-chart/convert-msaxlsx-to-html.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+if [ $# -ne 1 ]; then
+     echo "usage: $0 <xlsx-filename>"
+     exit 1
+fi
+
+# https://github.com/dilshod/xlsx2csv (this gracefully handles unicode nastiness, thankfully).
+pip install xlsx2csv
```

Comment:
`pip install` will require sudo for many setups.  Perhaps the script can check if the command exists, and exit with an appropriate warning if it doesn't.  I imagine the warning would look something like `'xlsx2csv' is required for this script to execute.  To install, run 'pip install xlsx2csv'.  Note that root access may be required to run pip commands.`